### DISCUSSION TITLE
Why do the metrics exposed by prometheus not include "workspace" label ?

### DISCUSSION THREAD

**Discussion by bluejam0303** [Kong/kong#11951] [Help] [answered]
Hi all,

I find the metrics exposed by prometheus not include "workspace" label 
If we have same route or service names on different workspaces, it would be hard to know the correct value.
Does anyone know why the metrics not include the workspace label on it ?

Thanks,
Jeffery

Thanks,
Jeffery

**Comment by StarlightIbuki** [comment, marked as answer]
Hi. Workspace is an enterprise feature. It looks like you're using an enterprise setup. You could reach out to customer support for help.

**Comment by bluejam0303** [comment]
Thanks , I will check it with support
